Runtimes/Core/cmake/modules/PlatformInfo.cmake

```diff
@@ -0,0 +1,51 @@
+if(NOT SwiftCore_SIZEOF_POINTER)
+  # This seems to be the most reliable way to actually get the size of a pointer
+  file(WRITE ${CMAKE_BINARY_DIR}/cmake/CompilerChecks/ptr_size "__SIZEOF_POINTER__")
+  set(size_check_command ${CMAKE_C_COMPILER} -x c -P -E -o - ${CMAKE_BINARY_DIR}/cmake/CompilerChecks/ptr_size)
+  if(CMAKE_C_COMPILER_TARGET)
+    list(APPEND size_check_command -target ${CMAKE_C_COMPILER_TARGET})
+  endif()
+  execute_process(
+    COMMAND ${size_check_command}
+    OUTPUT_VARIABLE ptr_bytes
+    OUTPUT_STRIP_TRAILING_WHITESPACE)
+  message(CONFIGURE_LOG "Stdlib Pointer size: ${ptr_bytes}")
+  set(SwiftCore_SIZEOF_POINTER "${ptr_bytes}" CACHE STRING "Size of a pointer in bytes")
+  mark_as_advanced(SwiftCore_SIZEOF_POINTER)
+endif()
+
+if(NOT SwiftCore_MODULE_TRIPLE)
+  # TODO: This logic should migrate to CMake once CMake supports installing swiftmodules
+  set(module_triple_command "${CMAKE_Swift_COMPILER}" -print-target-info)
+  if(CMAKE_Swift_COMPILER_TARGET)
+    list(APPEND module_triple_command -target ${CMAKE_Swift_COMPILER_TARGET})
+  endif()
+  execute_process(COMMAND ${module_triple_command} OUTPUT_VARIABLE target_info_json)
+  message(CONFIGURE_LOG "Swift target info: ${module_triple_command}\n"
+  "${target_info_json}")
+  string(JSON module_triple GET "${target_info_json}" "target" "moduleTriple")
+  set(SwiftCore_MODULE_TRIPLE "${module_triple}" CACHE STRING "swift module triple used for installed swiftmodule and swiftinterface files")
+  mark_as_advanced(SwiftCore_MODULE_TRIPLE)
+endif()
+
+if(NOT SwiftCore_SWIFTC_CLANGIMPORTER_RESOURCE_DIR)
+  # TODO: We need to separate the concept of compiler resources and the stdlib.
+  #       Compiler-resources in the compiler-resource directory are specific to
+  #       a given compiler. The headers in `lib/clang/include` and
```

Comment:
Nit: I might be thinking of other directory, but I think it is normally `lib/clang/<version>/include`.